src/core/test_helpers/kbn_server.ts

```diff
@@ -33,7 +33,11 @@ const DEFAULTS_SETTINGS = {
     port: 0,
     xsrf: { disableProtection: true },
   },
-  logging: { silent: true },
+  logging: {
+    root: {
+      level: 'off',
+    },
+  },
```

Comment:
I wonder if we should change `logging.root.level` to `all` in `createRootWithSettings` if the provided `cliArgs.verbose` is `true` to mimic the `cli/serve` behavior? OTOH all tests are passing without it, so it may not be that necessary.

Reply:
I don't think we should. It adds a lot of unnecessary noise for both developers running tests locally and CI. Plus we will have to pay 💵  for additional logs disk usage on CI.